compiler/src/test/scala/org/scalaexercises/exercises/compiler/CompilerSpec.scala

```diff
@@ -9,6 +9,38 @@ import scala.tools.nsc.{ Global, Settings }
 
 class CompilerSpec extends FunSpec with Matchers {
 
+  val metaInfoCode = """
+      import scala.Predef._
+
+      /** This object was generated by sbt-buildinfo. */
+      case object LibMetaInfo extends org.scalaexercises.definitions.BuildInfo {
+        /** The value is "exercises-stdlib". */
+        val name: String = "exercises-stdlib"
+        /** The value is "0.2.3-SNAPSHOT". */
+        val version: String = "0.2.3-SNAPSHOT"
+        /** The value is "2.11.7". */
+        val scalaVersion: String = "2.11.7"
+        /** The value is "org.scala-exercises". */
+        val organization: String = "org.scala-exercises"
+        /** The value is scala.collection.Seq("sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots", "sonatype-releases: https://oss.sonatype.org/content/repositories/releases"). */
+        val resolvers: scala.collection.Seq[String] = scala.collection.Seq("sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots", "sonatype-releases: https://oss.sonatype.org/content/repositories/releases")
+        /** The value is scala.collection.Seq("org.scala-lang:scala-library:2.11.7", "org.scala-lang:scala-compiler:2.11.7:ensime-internal", "org.scala-lang:scala-library:2.11.7:ensime-internal", "org.scala-lang:scala-reflect:2.11.7:ensime-internal", "org.scala-lang:scalap:2.11.7:ensime-internal", "com.chuusai:shapeless:2.2.5", "org.scalatest:scalatest:2.2.4", "org.scala-exercises:exercise-compiler:0.2.3-SNAPSHOT", "org.scala-exercises:definitions:0.2.3-SNAPSHOT", "org.scalacheck:scalacheck:1.12.5", "com.github.alexarchambault:scalacheck-shapeless_1.12:0.3.1"). */
+        val libraryDependencies: scala.collection.Seq[String] = scala.collection.Seq("org.scala-lang:scala-library:2.11.7", "org.scala-lang:scala-compiler:2.11.7:ensime-internal", "org.scala-lang:scala-library:2.11.7:ensime-internal", "org.scala-lang:scala-reflect:2.11.7:ensime-internal", "org.scala-lang:scalap:2.11.7:ensime-internal", "com.chuusai:shapeless:2.2.5", "org.scalatest:scalatest:2.2.4", "org.scala-exercises:exercise-compiler:0.2.3-SNAPSHOT", "org.scala-exercises:definitions:0.2.3-SNAPSHOT", "org.scalacheck:scalacheck:1.12.5", "com.github.alexarchambault:scalacheck-shapeless_1.12:0.3.1")
```

Comment:
What I'm saying is that it's a bug if the evaluator endpoint needs more than just the name of the artifact. For example to evaluate an exercise in the shapeless lib you just need to send the dependency for "org.scala-exercises" % "shapeless_2.11" % "1.0". The Evaluator server should automatically read the transitive dependencies and find shapeless and others that the exercise lib depends on. It's that not the case @juanpedromoreno @dialelo ?